src/components/Tooltip/Tooltip.tsx

```diff
@@ -18,18 +18,18 @@ export const TooltipStylesKey = 'ChromaTooltip';
 export const useStyles = makeStyles(
   (theme) => ({
     root: {
-      backgroundColor: theme.palette.common.white,
+      backgroundColor: theme.palette.common.black,
       borderRadius: theme.pxToRem(4),
       boxShadow: theme.boxShadows.tooltip,
-      color: theme.palette.text.hint,
+      color: theme.palette.common.white,
       fontFamily: theme.typography.fontFamily,
       fontSize: fontSizes.tooltip,
       letterSpacing: '0.021875em',
       maxWidth: theme.pxToRem(500),
       paddingLeft: theme.spacing(1),
       paddingRight: theme.spacing(1),
       paddingTop: theme.spacing(0.5),
-      paddingBottom: theme.spacing(0.5),
+      paddingBottom: theme.spacing(0.75),
```

Comment:
Out of curiosity, is the bottom padding more than the top for some visual fix?

Reply:
Yes, trying to counter balance the crazy amount of extra top space Lato arbitrarily adds.

Anxiously awaiting this: https://css-tricks.com/leading-trim-the-future-of-digital-typesetting/

Reply:
Haha nice no prob was just curious. Thanks!